.travis.yml

```diff
@@ -32,42 +32,51 @@ matrix:
   - env: PYTHON_VERSION=3 PYTHONIC=no  SYMPY_VER=1.4 OCT=4.4   DOCTEST=yes COLUMNS=80
   - env: PYTHON_VERSION=3 PYTHONIC=no  SYMPY_VER=1.4 OCT=5     DOCTEST=yes COLUMNS=80
   - env: PYTHON_VERSION=3 PYTHONIC=no  SYMPY_VER=1.4 OCT=devel DOCTEST=no  COLUMNS=80
+  - env: PYTHON_VERSION=3 PYTHONIC=yes SYMPY_VER=1.4 OCT=4.4   DOCTEST=yes COLUMNS=80
+  - env: PYTHON_VERSION=2 PYTHONIC=yes SYMPY_VER=1.4 OCT=4.4   DOCTEST=yes COLUMNS=80
+  - env: PYTHON_VERSION=3 PYTHONIC=yes SYMPY_VER=1.4 OCT=5     DOCTEST=yes COLUMNS=80
+  - env: PYTHON_VERSION=2 PYTHONIC=yes SYMPY_VER=1.4 OCT=5     DOCTEST=yes COLUMNS=80
   - env: PYTHON_VERSION=3 PYTHONIC=yes SYMPY_VER=1.4 OCT=devel DOCTEST=no  COLUMNS=80
-  - env: PYTHON_VERSION=3 PYTHONIC=yes SYMPY_VER=1.4 OCT=5     DOCTEST=no  COLUMNS=80
   - env: PYTHON_VERSION=2 PYTHONIC=yes SYMPY_VER=1.4 OCT=devel DOCTEST=no  COLUMNS=80
-  - env: PYTHON_VERSION=2 PYTHONIC=yes SYMPY_VER=1.4 OCT=5     DOCTEST=no  COLUMNS=80
+  - env: PYTHON_VERSION=3 PYTHONIC=dev SYMPY_VER=1.4 OCT=devel DOCTEST=no  COLUMNS=80
+  - env: PYTHON_VERSION=2 PYTHONIC=dev SYMPY_VER=1.4 OCT=devel DOCTEST=no  COLUMNS=80
+  - env: PYTHON_VERSION=3 PYTHONIC=dev SYMPY_VER=1.4 OCT=5     DOCTEST=yes COLUMNS=80
+  - env: PYTHON_VERSION=2 PYTHONIC=dev SYMPY_VER=1.4 OCT=5     DOCTEST=yes COLUMNS=80
   allow_failures:
   - env: PYTHON_VERSION=3 PYTHONIC=no  SYMPY_VER=1.4 OCT=devel DOCTEST=no  COLUMNS=80
   - env: PYTHON_VERSION=3 PYTHONIC=yes SYMPY_VER=1.4 OCT=devel DOCTEST=no  COLUMNS=80
-  - env: PYTHON_VERSION=3 PYTHONIC=yes SYMPY_VER=1.4 OCT=5     DOCTEST=no  COLUMNS=80
   - env: PYTHON_VERSION=2 PYTHONIC=yes SYMPY_VER=1.4 OCT=devel DOCTEST=no  COLUMNS=80
-  - env: PYTHON_VERSION=2 PYTHONIC=yes SYMPY_VER=1.4 OCT=5     DOCTEST=no  COLUMNS=80
+  - env: PYTHON_VERSION=3 PYTHONIC=dev SYMPY_VER=1.4 OCT=devel DOCTEST=no  COLUMNS=80
+  - env: PYTHON_VERSION=2 PYTHONIC=dev SYMPY_VER=1.4 OCT=devel DOCTEST=no  COLUMNS=80
+  - env: PYTHON_VERSION=3 PYTHONIC=dev SYMPY_VER=1.4 OCT=5     DOCTEST=yes COLUMNS=80
+  - env: PYTHON_VERSION=2 PYTHONIC=dev SYMPY_VER=1.4 OCT=5     DOCTEST=yes COLUMNS=80
 
 before_install:
 - docker pull mtmiller/octave:${OCT}
-- docker run --name=oc --detach --init --env=LC_ALL=C.UTF-8 --env=PYTHON_VERSION=${PYTHON_VERSION} --volume=$PWD:/octsympy:z mtmiller/octave:${OCT} sleep inf
+- docker run --name=oc --detach --init --env=LC_ALL=C.UTF-8 --env=PYTHON_VERSION=${PYTHON_VERSION} --env=PYTHON=python${PYTHON_VERSION} --volume=$PWD:/octsympy:z mtmiller/octave:${OCT} sleep inf
 
 install:
 - docker exec oc pip${PYTHON_VERSION} install sympy==${SYMPY_VER}
 - if [ "x$DOCTEST" = "xyes" ]; then
       docker exec oc octave --eval "pkg install -forge doctest";
   fi
 - if [ "x$PYTHONIC" = "xyes" ]; then
+      docker exec oc octave --eval "pkg install https://octave-pythonic-downloads.s3.amazonaws.com/releases/octave-pythonic-0.0.1.tar.gz";
```

Comment:
We could pull the package version out into a variable, but probably not worth it